index.html

```diff
@@ -0,0 +1,165 @@
+            </ul>
+            <div class="camp">
+              <a href="#">Campaign</a>
+            </div>
+          </div>
+        </div>
+
+        <nav class="showMenu">
+          <div class="logo-center">
+            <a href="index.html">
+              <img
+                class="m-logo"
+                src="./images/logo.ico"
+                alt="Company's Logo"
+              />
+            </a>
+          </div>
+          <div class="links">
+            <i class="fa fa-times fa-2x" aria-hidden="true"></i>
+            <ul>
+              <li><a href="about.html">About</a></li>
+              <li><a href="#">Progress</a></li>
+              <li><a href="#">Join</a></li>
+              <li><a href="#">Sponsor</a></li>
+              <li><a href="#">News</a></li>
+            </ul>
+          </div>
+          <div class="camp">
+            <a href="#">Campaign</a>
+          </div>
+        </nav>
+      </header>
+
+      <!-- Banner Section -->
+      <section class="describe">
+        <i id="hamburger" class="fa fa-bars fa-2x" aria-hidden="true"></i>
+        <h1>Eradicate cholera from the sixteen regions of Ghana</h1>
+        <p>
+          Cholera remains a global threat to public health and an indicator of
+          inequity and lack of social development. This forum seeks to address
+          key symptoms, treatment and ways it can be annihilated from the
+          country.
+        </p>
+        <div class="span-align">
+          <h2>2023.03.24(MON) ~ 28(FRI)</h2>
+          <span
+            ><img src="./images/Location.svg" alt="Location" /> International
+            Conference Hall, Ridge</span
+          >
+        </div>
+      </section>
+
+      <!-- Program Outline -->
+      <section class="main-row">
+        <h2 class="main-header">Main Program</h2>
+
+        <div class="main-program">
+          <div class="main-details">
+            <img src="./images/incoming_data.svg" alt="Causes image" />
+            <h3>Causes</h3>
+            <p>
+              The risk of a cholera epidemic is highest when poverty, war or
+              natural disasters force people to live in crowded conditions
+              without adequate sanitation.
+            </p>
+          </div>
+          <div class="main-details">
+            <img src="./images/shaking.svg" alt="Symptoms image" />
+            <h3>Symptoms</h3>
+            <p>
+              Symptoms cause mild or moderate diarrhea are often hard to tell.
+              Others develop more-serious signs usually within a few days of
+              infection.
+            </p>
+          </div>
+          <div class="main-details">
+            <img src="./images/treatment.svg" alt="Treatments image" />
+            <h3>Treatments</h3>
+            <p>
+              Cholera patients should be evaluated and treated quickly. With
+              proper treatment, even severely ill patients can be saved.
+            </p>
```

Comment:
- [ ] In the mobile version some of the `<h3>` heading values are overlapping with the `<p>` content. Please make sure to adjust the sizes each `.main-details` div so that the layout is nicely spaced just like in the cindy shins design

| Cindy Shin design    | Your design |
| ------------- | ------------- |
| <img width="327" alt="Screenshot 2023-04-21 at 1 11 57 AM" src="https://user-images.githubusercontent.com/44798044/233477382-d4f2cb2e-4783-4234-bd61-e52020fcb32f.png">| <img width="254" alt="Screenshot 2023-04-20 at 11 35 35 PM" src="https://user-images.githubusercontent.com/44798044/233477435-857b16ee-4abd-43ec-95f2-a3c0f72ecdd7.png">
 |

- [ ] In the desktop version you are missing some cards. There are five cards in the design and in your website there are four. Kindly add the missing number of cards into your website


| Cindy Shin design    | Your design |
| ------------- | ------------- |
| <img width="1033" alt="Screenshot 2023-04-20 at 11 41 39 PM" src="https://user-images.githubusercontent.com/44798044/233478066-5925fa0b-bfaa-42f9-86e7-1289e5bd6c3e.png">| <img width="1508" alt="Screenshot 2023-04-20 at 11 37 37 PM" src="https://user-images.githubusercontent.com/44798044/233478178-c4169fdc-20dc-4b6a-9133-1a06cadcb8a1.png">|

Reply:
- [ ] You did a good job working on this requirement, however, there's still some overlapping between the image and the title, I would suggest that you reduce the font size as 18px seems too large for mobile devices! 🤓 🔧 

|Template|Your design|
|---|---|
| ![image](https://user-images.githubusercontent.com/77069011/233646962-731362e0-e5d1-4d64-975c-e89036647a4e.png) | ![image](https://user-images.githubusercontent.com/77069011/233646816-c5591bd3-220d-4b84-a660-261a6125aaf8.png) |

- [ ] **[OPTIONAL]** I would also suggest that you reduce the horizontal spacing that each card has with the edges of the device, as shown in the template design! 🤓 🔧 


- [ ] **[OPTIONAL]** For the desktop version, I suggest that you have consistent card sizes, currently, each card has a different width, which ends up in a not-very attractive UI. Please consider applying this change! 🤓 🔧

|Template|
|---|
| ![image](https://user-images.githubusercontent.com/77069011/233648051-4e052d2e-7a0d-4f79-94a6-fe09b48ba7c8.png) |

|Your design|
|---|
| ![image](https://user-images.githubusercontent.com/77069011/233647669-8761eec2-a6cc-4564-a6d3-ab4ee8ba57ac.png) |